images/golang/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM bootstrap
+FROM bootstrap-amd64
```

Comment:
Don't we need the `ARCH` here? If we don't we would just create a regular amd64 golang image and could thus replace the `publish_multiarch_image.sh` for golang with the regular `publish_image.sh` no?

Reply:
We need multi-arch golang image, I have update the PR.